docs/basics/env-var-info.md

```diff
@@ -1,147 +1,161 @@
 
 ## DOCKERSHAREDDIR
-
 * System Detected value: `~/.config/appdata/shared`
+
 * Default value: `~/.config/appdata/shared`
+
 Description: This is the default shared folder between all containers. You can place anything here and it can be accessed from any other container, such as scripts.
 
 ## DOWNLOADSDIR
-
 * System Detected value: `~/Downloads`
+
 * Default value: `/mnt/downloads`
+
 Description: This is the default folder where all your downloads are located. To be used with SABnzbd, NZBGet, and any torrent clients.
 
 ## MEDIADIR_AUDIOBOOKS
-
 * System Detected value: `~/Audioooks`
+
 * Default value: `/mnt/medialibrary/audiobooks`
+
 Description: This is the default folder where all audiobooks are stored.
 
 ## MEDIADIR_BOOKS
-
 * System Detected value: `~/Books`
+
 * Default value: `/mnt/medialibrary/books`
+
 Description: This is the default folder where all books are stored.
 
 ## MEDIADIR_COMICS
-
 * System Detected value: `~/Comics`
+
 * Default value: `/mnt/medialibrary/comics`
+
 Description: This is the default folder where all comics are stored.
 
 ## MEDIADIR_MOVIES
-
 * System Detected value: `~/Movies`
+
 * Default value: `/mnt/medialibrary/movies`
+
 Description: This is the default folder where all movies are stored. To be used with Radarr, Bazarr, etc.
 
 ## MEDIADIR_MUSIC
-
 * System Detected value: `~/Music`
+
 * Default value: `/mnt/medialibrary/music`
+
 Description: This is the default folder where all movies are stored. To be used with Lidarr.
 
 ## MEDIADIR_TV
-
 * System Detected value: `~/TV`
+
 * Default value: `/mnt/medialibrary/tv`
+
 Description: This is the default folder where all movies are stored. To be used with Sonarr, Bazarr, etc.
 
 ## PGID
-
 * System Detected value: Detects the ID of your group
+
 * Default value: `1000` because this is the most common default on supported OS
+
 Description: This value can be obtained by using `id $user`.
 
 ## PUID
-
 * System Detected value: Detects the ID of your user
+
 * Default value: `1000` because this is the most common default on supported OS
+
 Description: This value can be obtained by using `id $user`.
 
 ## TZ
-
 * System Detected value: Uses the value found in `/etc/timezone`
+
 * Default value: `America/Chicago` because that's @nemchik's timezone (CST)
 
 ## LAN_NETWORK
+* System Detected value: Detects your local IP range.
+
+* Default value: `192.168.x.x/24`
 
-* System Detected value: Determined by your network interface
-* Default value: `192.168.x.x/24` if this value is blank or contains `x` DockSTARTer will automatically replace it with the System Detected value
 Description: Only accepts values in these ranges 10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16.
 
 ## NS1
-
 * Default value: `1.1.1.1`
+
 Description:
 
 ## NS2
-
 * Default value: `8.8.8.8`
+
 Description:
 
 ## VPN_ENABLE
-
 * Default value: `yes`
+
 Description:
 
 ## VPN_OPTIONS
+* Default value:
 
-* Default value: No default value
```

Comment:
I think we should in some way indicate when a variable is empty by default. Wording can be tweaked.